service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/c2uploaded/C2UploadedTemplate.java

```diff
@@ -5,11 +5,14 @@
 import lombok.experimental.SuperBuilder;
 import uk.gov.hmcts.reform.fpl.model.notify.SharedNotifyTemplate;
 
+import java.util.List;
+
 @EqualsAndHashCode(callSuper = true)
 @Data
 @SuperBuilder
 public final class C2UploadedTemplate extends SharedNotifyTemplate {
```

Comment:
I should probably create something like a "AdditionalOrdersUploadedTemplate" and use it in the appropriate places rather than using this class.